kolibri/plugins/registry.py

```diff
@@ -48,14 +67,15 @@ def initialize():
             try:
                 plugin_module = importlib.import_module(app + ".kolibri_plugin")
                 logger.debug("Loaded kolibri plugin: {}".format(app))
+                all_classes = dict([(name, cls) for name, cls in plugin_module.__dict__.items() if isinstance(cls, type)])
                 plugin_classes = []
-                for obj in plugin_module.PLUGINS:
+                for obj in all_classes.values():
                     if type(obj) == type and issubclass(obj, KolibriPluginBase):
                         plugin_classes.append(obj)
                 for plugin_klass in plugin_classes:
-                    plugin_obj = plugin_klass()
-                    for hook, callback in plugin_obj.hooks().items():
-                        hooks.register_hook(hook, callback)
+                    # Initialize the class, nothing more happens for now.
+                    logger.debug("Initializing plugin: {}".format(plugin_klass.__name__))
+                    plugin_klass()
```

Comment:
It also seems like you might only want to "initialize" non-abstract hook classes. Though I am having trouble supporting that statement, as I don't have a good intuition for the use of this event.